### DISCUSSION TITLE
Modifying SOC side links

### DISCUSSION THREAD

**Discussion by darth-drew** [Security-Onion-Solutions/securityonion#3664] [Unsupported Versions] [answered]
I am seeing the below errors when attempting to add custom links to the SOC side panel. I have tried a couple different tests, including copying the so-navigator line and pasting it at the end. Same behavior. Currently running 2.3.40 distributed airgap deployment. Is anyone else seeing this on their end?

![image](https://user-images.githubusercontent.com/12967094/112860654-03460200-9082-11eb-9503-ad68620ad92e.png)

**Comment by Masaya-A** [comment, marked as answer]
A comma (,) is needed at the end of the toolNavigator line.

**Reply by darth-drew** [reply]
Well obviously I put a... Nope, that was dumb. Comma was the issue. Oversight on my part. Thanks!
